curriculum/challenges/english/01-responsive-web-design/basic-css/use-rgb-values-to-color-elements.md

```diff
@@ -43,7 +43,7 @@ Your `body` element should have a black background.
 assert($('body').css('background-color') === 'rgb(0, 0, 0)');
 ```
 
-You should use `rgb` to give your `body` element a color of black.
+You should use `rgb` to give your `body` element a background-color of black.
```

Comment:
Since the background is what we want changed, I suggest changing this to simply "background", like the example used.
```suggestion
You should use `rgb` to give your `body` element a background of black.
```

Reply:
What example @RandellDawson? Not sure I see it. I think I would be fine with `background-color`.

Reply:
I see - Yea, I think I'm in favor of it then. Either way works for me.